#! /bin/sh /usr/share/dpatch/dpatch-run
## 30-fix-Makefile.dpatch by Sven Mueller <debian@incase.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Adjust Makefile to Debian's needs

@DPATCH@
diff -urNad trunk/Makefile /tmp/dpep.g7kFlz/trunk/Makefile
--- trunk/Makefile	2005-11-03 20:24:21.000000000 +0100
+++ /tmp/dpep.g7kFlz/trunk/Makefile	2005-11-03 20:29:10.154385505 +0100
@@ -14,43 +14,39 @@
 INSTALL:="/usr/bin/install"
 LN:="ln"
 
-.PHONY: all install uninstall clean
+.PHONY: all install clean
 all: spampd.8.gz spampd.html
 
 install: spampd.8.gz spampd.html
 	$(INSTALL) -D -m 755 spampd ${BINDIR}/spampd
-	$(INSTALL) -D -m 644 spampd.default ${ETCDIR}/default/spampd	
-	$(INSTALL) -D -m 755 spampd-init.sh $(DESTDIR)${INITDIR}/spampd
-	$(INSTALL) -D -m 644 spampd.html ${DOCDIR}/spampd.html
-	$(INSTALL) -D -m 644 changelog.txt ${DOCDIR}/changelog
-	for i in ${RUNLEVELDIRS}; do \
-		$(LN) -sf ${INITDIR}/spampd $i/S20spampd ; \
-	done
+#	$(INSTALL) -D -m 644 defaults ${ETCDIR}/default/spampd	
+#	$(INSTALL) -D -m 755 spampd-init.sh $(DESTDIR)${INITDIR}/spampd
+#	$(INSTALL) -D -m 644 spampd.html ${DOCDIR}/spampd.html
+#	$(INSTALL) -D -m 644 changelog.txt ${DOCDIR}/changelog.txt
+#	for i in ${RUNLEVELDIRS}; do \
+#		$(LN) -sf ${INITDIR}/spampd $i/S20spampd ; \
+#	done
 	$(INSTALL) -D -m 644 spampd.8.gz ${MANDIR}/man8/spampd.8.gz
 
-uninstall:
-	rm -f ${BINDIR}/spampd
-	rm -f ${ETCDIR}/default/spampd
-	rm -f $(DESTDIR)${INITDIR}/spampd
-	rm -f ${DOCDIR}/spampd.html
-	rm -f ${DOCDIR}/changelog
-	rmdir ${DOCDIR} || true
-	for i in ${RUNLEVELDIRS}; do \
-		rm -f $i/S20spampd ; \
-	done
-	rm -f ${MANDIR}/man8/spampd.8.gz
-
 spampd.8.gz: spampd.8
 	gzip -9 < spampd.8 > spampd.8.gz
 
 spampd.8: spampd
+	if [ -f spampd.8 ] && [ ! -f spampd.8.orig ] ; then mv -f spampd.8 spampd.8.orig ; fi
 	pod2man --section=8 --center="Spam Proxy Daemon" spampd > spampd.8
 
 spampd.html: spampd
+	if [ -f spampd.html ] && [ ! -f spampd.html.orig ] ; then mv -f spampd.html spampd.html.orig ; fi
 	pod2html --outfile spampd.html --header --norecurse --backlink '[Back to top]' --infile spampd
 	rm -f pod2htm?.tmp
 
 clean:
 	rm -f spampd.8.gz
+	[ -f spampd.html.orig] && mv -f spampd.html.orig spampd.html
+	[ -f spampd.8.orig] && mv -f spampd.8.orig spampd.8
+
+realclean:
+	rm -f spampd.8.gz
 	rm -f spampd.8
 	rm -f spampd.html
+	rm -f spampd*.orig
